firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/metadata/RolloutAssignmentList.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.crashlytics.internal.metadata;
+
+import com.google.firebase.crashlytics.internal.Logger;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/** Handles RolloutsState for metadata */
+public class RolloutAssignmentList {
```

Comment:
I only see this being used in tests - is this being used yet?

Reply:
Nope, just prepare for integration, this class will eventually add to UserMetaData.